src/app/routes/feed/stage.tsx

```diff
@@ -301,16 +334,7 @@ export default function FeedStage(): ReactNode {
 			getItemId={ ( item: FeedPost ): string => item.id.toString() }
 			selection={ selection }
 			onChangeSelection={ changeSelection }
-			empty={
-				<p>
-					{ normalizedView.search || ( normalizedView.filters && normalizedView.filters.length > 0 )
-						? __( 'No posts found.', 'activitypub' )
-						: __(
-								'No posts found in your feed. Posts from ActivityPub actors you follow will appear here.',
-								'activitypub'
-						  ) }
-				</p>
-			}
+			empty={ <p>{ renderEmptyStateContent() }</p> }
```

Comment:
```suggestion
			empty={ <EmptyState /> }
```
We should be able to pass a zero-prop component here.